### PR TITLE
shortenURL and getNewsFeed endpoint

### LINKED ISSUE #19: Communications - shortenURL endpoint
Endpoint details: https://api.helioviewer.org/docs/v2/api/api_groups/official_clients.html#shortenurl

### LINKED ISSUE #20: Communications - getNewsFeed endpoint
Endpoint details: https://api.helioviewer.org/docs/v2/api/api_groups/official_clients.html#shortenurl

### PR DESCRIPTION
**Pull request by akash5100**
* closes #19 
* closes #20

### CODE REVIEW

File: hvpy/api_groups/communications/tests/test_shorten_url.py [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+from hvpy.api_groups.communications.shorten_url import shortenURLInputParameters
+
+
+def test_string_response():
+    response = shortenURL(
+        queryString="https://api.helioviewer.org/v2/queueMovie/?startTime=2010-03-01T12:12:12Z&endTime=2010-03-04T12:12:12Z"
+    )
+    assert isinstance(response, dict)
+
+    print(response)
```
Comment (nabobalis) [inline hvpy/api_groups/communications/tests/test_shorten_url.py:11]:
```suggestion
```

File: hvpy/api_groups/communications/tests/test_shorten_url.py [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+
```
Comment (nabobalis) [inline hvpy/api_groups/communications/tests/test_shorten_url.py:12]:
```suggestion
```

File: hvpy/facade.py [hunk 3]
```diff
@@ -474,3 +476,49 @@ def downloadMovie(
...
+        callback=callback,
+    )
+    return execute_api_call(input_parameters=params)
+
+
+@add_shared_docstring(shortenURLInputParameters)
+def shortenURL(
+    queryString: str,
+    callback: Optional[str] = None,
+) -> Union[bytes, str, Dict[str, Any]]:
```
Comment (nabobalis) [inline hvpy/facade.py:507]: Does it return bytes?

File: hvpy/facade.py [hunk 4]
```diff
@@ -474,3 +476,49 @@ def downloadMovie(
         hq=hq,
     )
     return execute_api_call(input_parameters=params)
+
+
+@add_shared_docstring(getNewsFeedInputParameters)
+def getNewsFeed(
+    callback: Optional[str] = None,
+) -> Union[bytes, str, Dict[str, Any]]:
```
Comment (nabobalis) [inline hvpy/facade.py:484]: Does it return bytes?

File: hvpy/facade.py (hunk 3 above)
Comment (akash5100) [reply hvpy/facade.py:507]: It does not but mypy throws an error: 
```
hvpy/facade.py:500: error: Incompatible return value type (got "Union[bytes, str, Dict[str, Any]]", expected "Union[str, Dict[str, Any]]")
hvpy/facade.py:524: error: Incompatible return value type (got "Union[bytes, str, Dict[str, Any]]", expected "Union[str, Dict[str, Any]]")
```

Comment (nabobalis) [reply hvpy/facade.py:507]: I wonder why, something to fix later on.

Comment (nabobalis) [reply hvpy/facade.py:507]: Oh I see why, thats ok.

**Review by nabobalis — APPROVED** [review]

**Pull request merged**